test/steps/v2_steps.py

```diff
@@ -1799,10 +1843,25 @@ def sign_transaction_with_signing_account(context):
     context.signed_transaction = context.transaction.sign(private_key)
 
 
+@then('the base64 encoded signed transactions should equal "{goldens}"')
+def compare_stxns_array_to_base64_golden(context, goldens):
+    golden_strings = goldens.split(",")
```

Comment:
Probably a good idea to compare lengths as well

Or just convert `context.signed_transactions` to an encoded array and compare that to `golden_strings`